department-of-veterans-affairs/caseflow
# SeanC/APPEALS-13786

[Resolves APPEALS-13786](https://vajira.max.gov/browse/APPEALS-13786)

### Description
Added a feature toggle to switch between having veteran ssn or participate id as a column in the table and showing a label above the search box on the decision review page.

### Acceptance Criteria
- [x] There is a feature toggle named decision_review_queue_ssn_column
- [x] When feature toggle is enabled the Veteran SSN column is visible in the table
- [x] When feature toggle is enabled the Veteran Participant Id column is not visible in the table
- [x] When feature toggle is enabled the label above the search box is visible
- [x] When feature toggle is disabled the Veteran Participant Id column is visible in the table
- [x] When feature toggle is disabled the Veteran SSN column is not visible in the table
- [x] When feature toggle is disabled the label above the search box is not visible
- [x] Feature toggle is added to storybook file
- [x] Affected tests are all passing

### Testing Plan
1. Log in as BVAISHAW
2. Go to /decision_reviews/vha
3. Look for label above the search box to not be visible
4. Look for Veteran Participant Id column, after the Claimant column and before the Issues column
5. Run command `make c` to open the rails console
6. In the rails console, run the command: `FeatureToggle.enable!(:decision_review_queue_ssn_column)` to enable the feature toggle
7. Refresh page
8. Look for Veteran SSN column, after the Claimant column and before the Issues column
9. Look for label above search box
10. In the rails console, run the command: `FeatureToggle.disable!(:decision_review_queue_ssn_column)` to disable the feature toggle
11. Refresh page
12. Label and Veteran SSN column should not be visible. The Veteran Participant Id column should be visible in the table.
